### PR TITLE
Enhance chart

### PR DESCRIPTION
**Pull request by QuantumEnigmaa**
Chart enhancements, especially for grafana-agent's config for pyroscope

### CODE REVIEW

**Review by hervenicol — COMMENTED** [review]
Is this something we want as default?
I mean, if a customer installs Pyroscope, is it something he will want by default, with this name?

If yes, fine.
If that's specific to our setup on our MCs, then this is not the place.

**Comment by QuentinBisson** [comment]
Also do we want thé agent from this chart instead of ours ?

**Comment by QuantumEnigmaa** [comment]
> Is this something we want as default?

> Also do we want thé agent from this chart instead of ours ?

Definitely not. Hopefully I will be able to use our `grafana-agent` instead of having to rely on this embedded one but I'm still working on it.

File: helm/pyroscope/values.yaml [hunk 1]
```diff
@@ -1,7 +1,38 @@
+pyroscopeagent:
```
Comment (hervenicol) [inline helm/pyroscope/values.yaml:1]: Maybe `pyroscopeAgent` (camelCase) would be easier to read?
BTW, the schema already uses camelCase.

Comment (QuantumEnigmaa) [reply helm/pyroscope/values.yaml:1]: That's what I was using a few minutes ago but upon deployment I was running into errors for sa's names that are not allowing camelCase for their names :/

**Comment by QuantumEnigmaa** [comment]
The handling of the `pyroscope-agent was tranfered towards `grafana-agent-app` : https://github.com/giantswarm/grafana-agent-app/pull/35

**Review by QuentinBisson — APPROVED** [review]
Maybe add a changelog entry ? Otherwise that's good

**Pull request merged**